Elastic indices information (#480)
ElrondNetwork/docs.elrond.com: docs/sdk-and-tools/indices/transactions.md

---

> id: transactions

Reviewer: as discussed, please set the `id` to `es-index-transactions`

Author: done

> title: transactions

Reviewer: Elasticsearch index: transactions

Author: done

---

> The `_id` field for this index is composed of transaction hash hex encoded.

Reviewer 2: ```suggestion
The `_id` field for this index is composed of hex encoded transaction hash.
```

Author: done

(example: `cad4692a092226d68fde24840586bdf36b30e02dc4bf2a73516730867545d53c`)

| Field             | Description                                                                                                                            |
|-------------------|----------------------------------------------------------------------------------------------------------------------------------------|

> | miniBlockHash     | The miniBlockHash represents the hash of the miniblocks in which the transaction is included.                                          |

Reviewer: miniblock*

Author: done

> | miniBlockHash     | The miniBlockHash represents the hash of the miniblock in which the transaction is included.                                                         |

Reviewer 2: ```suggestion
| miniBlockHash     | The miniBlockHash represents the hash of the miniblock in which the transaction was included.                                                         |
```

Author: done

> | nonce             | The nonce field represents sequence number of transactions sent from a given address.                                                  |

Author: done

> | nonce             | The nonce field represents the nonce of the sender at the transaction's time.                                                                        |

Reviewer 2: ```suggestion
| nonce             | The nonce field represents the transactions' number sent from the given address.                                                                        |
```

> | round             | The round field represents the round of block when transaction is executed.                                                            |

Author: done

> | round             | The round field represents the round of the block when the transaction is executed.                                                                  |

Reviewer 2: ```suggestion
| round             | The round field represents the round of the block when the transaction was executed.                                                                  |
```

Author: done

> | value             | The value field represents the amount of EGLD from the sender to the receiver.                                                         |

Reviewer: The value field represents the number of EGLD to be sent from the sender to the receiver

Author: done

| receiver          | The receiver field represents the destination address of the transaction.                                                              |

> | sender            | The sender field field represents the address of the transaction sender.                                                               |

Author: done

| receiverShard     | The receiverShard field represents the shard ID of the receiver address.                                                               |
| senderShard       | The senderShard field represents the shard ID of the sender address.                                                                   |

> | gasPrice          | The gasPrice field represents the price of the gas.                                                                                    |

Reviewer: The gasPrice field represents the amount to be paid for each gas unit

Author: done

> | gasLimit          | The gasLimit field represents the limit of the amount of EGLD the sender is willing to pay for the transaction.                        |                                                                  |

Reviewer: The gasLimit field represents the maximum gas units the sender is willing to pay for.

Author: done.

| gasUsed           | The gasUsed field represents the amount of gas used by the transaction.                                                                |
| fee               | The fee field represents the amount of EGLD the sender paid for the transaction.                                                       |
| initialPaidFee    | The initialPaidFee field represents the initial amount of EGLD the sender paid for the transaction, before the refund.                 |

> | data              | The data field holds additional information for a transaction. It can either contain a simple message/ESDT related information/etc.    |

Reviewer: It can contain a simple message, a function call, an ESDT transfer payload, or so on

> | signature         | The signature field represents the digital information used in cryptocurrency transactions to verify the identity of its participants. |

Reviewer: The signature obtained by the sender after signing the transaction. It is hex-encoded

Author: done

| timestamp         | The timestamp field represents the timestamp of the block in which the transaction was executed.                                       |
| status            | The status field represents the status of the transaction.                                                                             |
| senderUserName    | The senderUserName field represents the username of the sender address.                                                                |
| receiverUserName  | The receiverUserName field represents the username of the receiver address.                                                            |
| hasScResults      | The hasScResults field is true if the transaction has smart contract results.                                                          |
| isScCall          | The isScCall field is true if the transaction is a smart contract call.                                                                |
| hasOperations     | The hasOperations field is true if the transaction has smart contract results or logs.                                                 |

> | tokens            | The tokens field contains list of ESDT tokens that are transferred based on the data field.                                            |

Reviewer: a list

Author: done

> | esdtValues        | The esdtValues field contains list of ESDT values that are transferred based on the data field.                                        |

Reviewer: a list

Author: done

> | receivers         | The receivers field contains list of receiver addresses in case of ESDTNFTTransfer or MultiESDTTransfer.                               |

Reviewer: a list

Author: done

> | receiversShardIDs | The receiversShardIDs field contains list of receiver addresses shard IDs.                                                             |

Reviewer: a list

Author: done

| type              | The type field represents the type of the transaction based on the data field.                                                         |
| operation         | The operation field represents the operation of the transaction based on the data field.                                               |

> | isRelayed         | The isRelayed field  is true if the transaction is a relayed transaction.                                                              |

Reviewer: double white spaces between `field` and `is`

Author: removed.

| version           | The version field represents the version of the transaction.                                                                           |

## Query examples

> #### Fetch the latest transactions for an address

Reviewer: please use only 3 `#`

Reviewer: for -> of*

Author: done
